### PR TITLE
Add hyperviscosity conversion for CAM-SE model

### PR DESCRIPTION
**Pull request by charleskawczynski**
This PR adds a conversion for the CAM-SE hyperviscosity. I plan to use this in the parameter sweeps.

### CODE REVIEW

File: src/solver/model_getters.jl [hunk 1]
```diff
@@ -45,6 +45,18 @@ function get_hyperdiffusion_model(parsed_args, ::Type{FT}) where {FT}
             κ₄_tracer,
             divergence_damping_factor,
         )
+    elseif hyperdiff_name in ("CAM_SE",)
+        # To match hyperviscosity coefficients in:
+        #    https://agupubs.onlinelibrary.wiley.com/doi/epdf/10.1029/2017MS001257
+        #    for equation A18 and A19
+        Ne = parsed_args["h_elem"]
+        κ₄_vorticity = FT(0.15 * (30 / Ne * 1.1 * 10^5)) # ν_vort
+        κ₄_tracer = FT(0.15 * (30 / Ne * 1.1 * 10^5)) # ν_q
```
Comment (tapios) [inline src/solver/model_getters.jl:54]:
```suggestion
        κ₄_tracer = FT(0.75 * (30 / Ne * 1.1 * 10^5)^3) # ν_q
```

**Review by tapios — COMMENTED** [review]
The paper also uses `divergence_damping_factor = 5'. 

@szy21 is implenting a hyperdiffusion formulation (based on the physical grid spacing) that will no longer need this. Depending on when she'll be done, we may not need this.

**Comment by szy21** [comment]
The grid-spacing-dependent hyperdiffusion needs [ClimaCore 739](https://github.com/CliMA/ClimaCore.jl/pull/739), so it will take slightly longer. If you need something now this is good.

**Comment by szy21** [comment]
I'll merge this in, as the ClimaCore PR for minimum nodal distance has not been merged yet. We can remove this later.

**Comment by szy21** [comment]
@akshaysridhar You can use this if you want to follow CAM hyperdiffusion.

**Comment by akshaysridhar** [comment]
Noted thanks!

**Pull request merged**